statistics/table.go

```diff
@@ -610,45 +733,46 @@ func (coll *HistColl) GenerateHistCollFromColumnInfo(tblInfo *model.TableInfo, c
 // Usually, we don't want to trigger stats loading for pseudo table.
 // But there are exceptional cases. In such cases, we should pass allowTriggerLoading as true.
 // Such case could possibly happen in getStatsTable().
-func PseudoTable(tblInfo *model.TableInfo, allowTriggerLoading bool) *Table {
-	const fakePhysicalID int64 = -1
+func PseudoTable(tblInfo *model.TableInfo, allowTriggerLoading bool, allowFillHistMeta bool) *Table {
```

Comment:
Please add some comments to `allowFillHistMeta`.

Reply:
Yes, it's a temp solution to add this new param. We meet the problem with the pseudo table's memory usage.
But fully refactoring needs to also change some estimation logic. After some hard trying, I removed those changes in the pull since this pull is already large enough.:joy:

Reply:
This pull reduces some part of the pseudo table's memory usage, but not fully resolve it🤣